src/sprite.rs

```diff
@@ -228,8 +228,18 @@ impl Spritesheet {
 }
 
 /// Returns the name (unique id within a spritesheet) taken from a file.
```

Comment:
Let's add an explanation of what `base_path` is used for

```suggestion
/// Returns the name (unique id within a spritesheet) taken from a file.
///
/// If `base_path` is given, it's used as prefix in the sprite name (minus any parents
/// shared with `path`).
```

Reply:
You're explanation starts with _"If `base_path` is given"_. There is not really a choice tho 😂 so I adjusted it a bit. 

Addressed in 7a6a857c1c630eeeefcf08bb992462a2f8afffa0